src/main/java/net/sf/jabref/model/entry/InternalBibtexFields.java

```diff
@@ -305,17 +273,29 @@ private InternalBibtexFields(boolean serializeSpecialFields) {
             add(field);
         }
 
-        // collect all public fields for the PUBLIC_FIELDS array
-        for (BibtexSingleField sField : fieldSet.values()) {
-            if (!sField.isPrivate()) {
-                PUBLIC_FIELDS.add(sField.getFieldName());
-                // or export the complete BibtexSingleField ?
-                // BibtexSingleField.toString() { return fieldname ; }
-            }
-        }
+    }
 
-        // sort the entries
-        Collections.sort(PUBLIC_FIELDS);
+
+    public static void updateTimeStampField(String timeStampFieldName) {
```

Comment:
No problems. I've just noted this trend and there is a risk that people get discouraged touching the really troublesome code.

What we should try to do though, since clearly most of the things pointed out are valid concerns, is to try to write them down somewhere, like https://github.com/koppor/jabref/issues for those rainy days... But I can admit that I'm not very good at it, even though I try from time to time.